backend/src/models/CampusHousingModel.ts

```
@@ -0,0 +1,58 @@
+  studentEmail: string;
+  isIdentityVerified: boolean;
+}
+
+export class CampusHousingModel {
+  public id: string;
+  public propertyTitle: string;
+  public streetAddress: string;
+  public leaseDuration: string;
+  public monthlyRentUSD: number;
+  public securityDepositUSD: number;
+  public roomLayout: 'Private Studio' | 'Private Room (Shared Bath)' | 'Entire Apartment';
```

ESLint

[error] Replace `'Private·Studio'·|·'Private·Room·(Shared·Bath)'·|·'Entire·Apartment'` with `"Private·Studio"·|·"Private·Room·(Shared·Bath)"·|·"Entire·Apartment"`

(prettier/prettier)